src/GrpcProtos/capture.proto

```diff
@@ -517,6 +517,23 @@ message GpuDebugMarker {
   Color color = 5;
 }
 
+message PresentEvent {
```

Comment:
How will we group the events? pid+tid+source? I would expect smth like `function_id` (e.g. `etw_id`). 

I mean, there might be multiple ETW events, and only some of them will be related to frame track, right?

Reply:
The PresentEvent only pertains to graphics ETW related to swapchain presentation, this is not a generic EtwEvent class. As such, those events will be grouped by their "Source", which currently can either be "D3d9" or "Dxgi". All received "PresentEvent" will be displayed on the frame track (well in some form, i.e. the interval between consecutive events).